### DISCUSSION TITLE
Error while ruuning yarn dsv command

### DISCUSSION THREAD

**Discussion by KT-Developer** [RocketChat/Rocket.Chat#33540] [Q&A]
@rocket.chat/meteor:dsv:    
@rocket.chat/meteor:dsv:    
@rocket.chat/meteor:dsv: => Your application has errors. Waiting for file change.
@rocket.chat/meteor:dsv:                                               
@rocket.chat/meteor:dsv: /home/kunsh/.meteor/packages/meteor-tool/.2.16.0.3vipvb.sgul9++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/mongodb/bin/mongod:
@rocket.chat/meteor:dsv: error while loading shared libraries: libcrypto.so.3: cannot open shared object
@rocket.chat/meteor:dsv: file: No such file or directory
@rocket.chat/meteor:dsv: Unexpected mongo exit code 127. Restarting.
@rocket.chat/meteor:dsv: 
@rocket.chat/meteor:dsv: /home/kunsh/.meteor/packages/meteor-tool/.2.16.0.3vipvb.sgul9++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/mongodb/bin/mongod:
@rocket.chat/meteor:dsv: error while loading shared libraries: libcrypto.so.3: cannot open shared object
@rocket.chat/meteor:dsv: file: No such file or directory
@rocket.chat/meteor:dsv: Unexpected mongo exit code 127. Restarting.
@rocket.chat/meteor:dsv: 
@rocket.chat/meteor:dsv: /home/kunsh/.meteor/packages/meteor-tool/.2.16.0.3vipvb.sgul9++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/mongodb/bin/mongod:
@rocket.chat/meteor:dsv: error while loading shared libraries: libcrypto.so.3: cannot open shared object
@rocket.chat/meteor:dsv: file: No such file or directory
@rocket.chat/meteor:dsv: Unexpected mongo exit code 127. Restarting.
@rocket.chat/meteor:dsv: Can't start Mongo server.
@rocket.chat/meteor:dsv: npm ERR! code ELIFECYCLE
@rocket.chat/meteor:dsv: npm ERR! errno 254
@rocket.chat/meteor:dsv: npm ERR! @rocket.chat/meteor@6.14.0-develop dev: `meteor --exclude-archs "web.browser.legacy, web.cordova"`
@rocket.chat/meteor:dsv: npm ERR! Exit status 254
@rocket.chat/meteor:dsv: npm ERR! 
@rocket.chat/meteor:dsv: npm ERR! Failed at the @rocket.chat/meteor@6.14.0-develop dev script.
@rocket.chat/meteor:dsv: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
@rocket.chat/meteor:dsv: 
@rocket.chat/meteor:dsv: npm ERR! A complete log of this run can be found in:
@rocket.chat/meteor:dsv: npm ERR!     /home/kunsh/.npm/_logs/2024-10-11T15_05_05_684Z-debug.log
@rocket.chat/meteor:dsv: 
@rocket.chat/meteor:dsv: ERROR: command finished with error: command (/home/kunsh/Rocket.Chat/apps/meteor) /tmp/xfs-4e032787/yarn run dsv exited (254)
@rocket.chat/meteor#dsv: command (/home/kunsh/Rocket.Chat/apps/meteor) /tmp/xfs-4e032787/yarn run dsv exited (254)

 Tasks:    48 successful, 49 total
Cached:    48 cached, 49 total
  Time:    8.664s 
Failed:    @rocket.chat/meteor#dsv

pls help me to resolve it 
Image attached for the same fir better readability 
![Screenshot from 2024-10-11 20-38-18](https://github.com/user-attachments/assets/91f0b6d0-fba1-43e8-be06-cafd251ff151)

**Comment by DhruvArvindSingh** [comment]
Hey does the issue got solved??

**Comment by reetp** [comment]
No response. Closing.

Please ask for help here:

https://open.rocket.chat/channel/support
